Gemfile

```diff
@@ -7,6 +7,9 @@ eval_gemfile local_gemfile if File.readable?(local_gemfile)
 # Specify your gem's dependencies in active_model_serializers.gemspec
 gemspec
 
+# TODO: remove when @beauby re-publishes this gem
+gem 'jsonapi', github: 'beauby/jsonapi'
```

Comment:
I think this would prevent a merge, right? Or at the very least we'd have to tell anyone running off of master (like me) to add a similar entry to their app's `Gemfile`.